crawlers/mooncrawl/eth_scan.py

```diff
@@ -0,0 +1,184 @@
+import argparse
+import boto3
+import csv
+import codecs
+import json
+import os
+from sqlalchemy.orm import Session
+from moonstreamdb.db import yield_db_session_ctx
+import sys
+import time
+from datetime import datetime
+from typing import Any, List, Optional, Tuple, Dict
+from dataclasses import dataclass
+from sqlalchemy.sql.expression import label, text
+from .version import MOONCRAWL_VERSION
+from moonstreamdb.models import EthereumAddress, EthereumLabel
+import requests
+
+
+ETH_SCAN_TOKEN = os.environ.get("MOONSTREAM_ETHERSCAN_TOKEN")
+if ETH_SCAN_TOKEN is None:
```

Comment:
Validation can live here. Other crawlers may not care if this token is set or not.